src/core/services/maps-api-loader/lazy-maps-api-loader.ts

```diff
@@ -1,4 +1,10 @@
-import {Inject, Injectable, Optional, Provider, provide} from '@angular/core';
+/**
```

Comment:
please remove this comment (as it gets added by the build process automatically)